ocdev/plugins/setup/setup.py

```diff
@@ -167,5 +167,11 @@ def clone_repo(self, url, branch='master', directory=None, no_history=False):
         if directory:
             cmd.append(directory)
 
-
-        return check_call(cmd)
\ No newline at end of file
+        assert not os.path.exists('{0}/core'.format(os.getcwd())), 'cannot clone, {0}/core already exists'.format(os.getcwd())
+        assert os.access(os.getcwd(), os.W_OK), 'Insufficient privilege to write to this directory'
```

Comment:
https://docs.python.org/3.4/library/subprocess.html#subprocess.check_call throws a CalledProcessError. Im wondering if the easiest fix just be to catch **Exception** instead of **PluginError** and render the error message there